### ISSUE TITLE
dateTimeStamp not supported

### ISSUE THREAD

**Issue by redbulll9** [lmu-bioinformatics/xmlpipedb#52]
Hi,

Is there no support for dateTimeStamp data type in this repository?
![image](https://user-images.githubusercontent.com/1351410/63854167-2c8c4100-c9ba-11e9-8479-91af697e642d.png)

This is the error message that i am getting

**Comment by dondi** [comment]
True, support may be missing for certain data types. The underlying library that we used for the initial XSD-to-database conversion, [HyperJAXB2](https://www.openhub.net/p/hyperjaxb2), is responsible for the initial conversion of the XSD into a schema and might not have had inherent support for `dateTimeStamp`.

The primary XSD/DTDs that we have worked with—UniProt and GO—do not include `dateTimeStamp` in their schemas and thus our knowledge of HyperJAXB2’s scope centers primarily on how it handles the UniProt and GO schemas. (which, for our purposes, have been sufficient)

It would indeed be disappointing if HyperJAXB2 did not support `dateTimeStamp` natively, but unfortunately that is a question for that project rather than this one.
